### ISSUE TITLE
Utilisation multiple d'Arduino impossible

### ISSUE THREAD

**Issue by NicolasD-62** [GladysProject/gladys-serial#4]
Dans le fichier "sendCode.js", on utilise systématiquement le premier port (`var port = ports[0];`).
Du coup, il est impossible d'utiliser plusieurs Arduinos.

L'idéal serait de passer en paramètre l'id du port à utiliser. Ce dernier pourrait être stocker lors de la création d'un device.

Note: Issue aussi crée dans [https://github.com/isokar/gladys-rflink](https://github.com/isokar/gladys-rflink)

**Comment by Pierre-Gilles** [comment]
Effectivement ! 

Quand tu dis "id du port à utiliser", on est d'accord on parle de "/dev/tty***" ?

**Comment by NicolasD-62** [comment]
Oui, ça peut être une idée, et vu que c'est lié au matériel, ça ne risque pas de changer.

**Comment by ForzAguanno** [comment]
Bonjour à vous @Pierre-Gilles et @NickDub 
Pouvez vous m'aider sur ce sujet svp ?
Je cherche à connecter plusieurs arduinos sur gladys en communication série usb.
A cause de ce 
var port = ports [0];
Je ne peux en contrôler qu'un seul..
